tests/testthat/test-quo.R

```diff
@@ -1,25 +1,81 @@
+# quo_c ----
+## Test 1: `quo_c` works in concatenating and indexing quosures ----
+test_that("quo_c Test 1: `quo_c` works in concatenating and indexing quosures", {
+  x <- quo(USUBJID)
+  y <- quo(STUDYID)
+
+  expect_equal(
+    expected = quo(USUBJID),
+    object = quo_c(x, NULL, y)[[1]]
+  )
+  expect_equal(
+    expected = quo(STUDYID),
+    object = quo_c(x, NULL, y)[[2]]
+  )
+})
+
+## Test 2: `quo_c` returns error if non-quosures are input ----
+test_that("quo_c Test 2: `quo_c` returns error if non-quosures are input", {
+  USUBJID <- "01-701-1015" # nolint
```

Comment:
how come this needs a nolint?

Reply:
I was getting this:
![image](https://user-images.githubusercontent.com/43018762/196957165-ad4b2584-021d-4c5d-aaa9-90b66daa24c8.png) 
I suppose I can just change it to avoid the nolint, my thought in using the capitalized format was to somewhat mimic the usual SDTM/ADaM data clinical trial programmers run into

Reply:
ah i see no problem if you want to keep as is